src/ActionButton/ActionButton.js

```diff
@@ -10,47 +16,51 @@ export default class WrappedActionButton extends Component {
     color: '#fff',
   }
 
-  renderSub() {
-    let { color, backgroundColor, icon, action } = this.props
+  renderPaper() {
```

Comment:
We should delete `renderSub` if we're no longer using it. If it's already deleted, we should rename this to be `renderSub`